os-client/src/main/kotlin/io/provenance/scope/objectstore/client/OsClient.kt

```diff
@@ -91,10 +91,10 @@ open class OsClient(
             }
 
         //Bind headers to stubs
-        objectAsyncClient = MetadataUtils.attachHeaders(ObjectServiceGrpc.newStub(channel), headers)
-        objectFutureClient = MetadataUtils.attachHeaders(ObjectServiceGrpc.newFutureStub(channel), headers)
-        publicKeyBlockingClient = MetadataUtils.attachHeaders(PublicKeyServiceGrpc.newBlockingStub(channel), headers)
-        mailboxBlockingClient = MetadataUtils.attachHeaders(MailboxServiceGrpc.newBlockingStub(channel), headers)
+        objectAsyncClient = ObjectServiceGrpc.newStub(channel).withInterceptors(MetadataUtils.newAttachHeadersInterceptor(headers))
```

Comment:
Hey, the thing that was preventing me from upgrading grpc in the bootstrap plugin